ui/src/App.tsx

```diff
@@ -71,7 +72,11 @@ export default function App() {
   const setCurFlowID = (id: string) => {
     curFlowID.current = id;
     setFlowID(id);
-    localStorage.setItem("curFlowID", id);
+    const workflow = getWorkflow(id);
+    if (workflow) {
+      localStorage.setItem("curFlowID", id);
+      // localStorage.setItem("comfy_workspace_workflow", workflow.json);
```

Comment:
thinking of saving a cache of json so don't need to wait for db loading to load last opened workflow....